thoth/python/source.py

```diff
@@ -414,38 +388,42 @@ def provides_package_version(self, package_name: str, package_version: str) -> b
             return False
 
     @lru_cache(maxsize=10)
-    def get_package_data(self, package_name: str, package_version: str, with_included_files: bool = False) -> list:
-        """Get information about release hashes and symbols available in this source index."""
+    def get_package_hashes(
+        self, package_name: str, package_version: str, with_included_files: bool = False, with_symbols: bool = False
+    ) -> list:
+        """Get information about release hashes available in this source index."""
         if self.warehouse:
             return self._warehouse_get_package_hashes(package_name, package_version, with_included_files)
```

Comment:
`with_symbols` is lost here

Reply:
it's not used by `_warehouse_get_package_hashes` right? did i miss something?

the only difference I noticed between `get_package_data` func and `get_package_hashes` func was the part 
`doc["symbols"] = artifact.get_versioned_symbols()`

thats the reason to include the param `with_symbols`
please correct me if i missed it somewhere

Reply:
Just took a look at the code. My worry was that the symbols were never added to the document if the flag was  set. It seems the opposite is actually true. `_warehouse_get_package_hashes` always adds the symbols not matter what. I think we should add a conditional statement in `_warehouse_get_package_hashes` so the return format is consistent. Let me know what you think

Reply:
ack, i have include the flag `with_symbols` in `_warehouse_get_package_hashes` as well.
Make sense to have consistency.

PTAL and Review again.
Thanks